eos-tech-support/eos-write-installer

```diff
@@ -27,23 +27,27 @@ eval set -- "$ARGS"
 usage() {
     cat <<EOF
 Usage:
-   $0 [options] DEVICE
+   $0 --os-image PATH --installer PATH DEVICE
+   $0 --latest DEVICE
+
 Arguments:
     DEVICE	Device path (e.g. '/dev/sdb' or '/dev/mmcblk0')
+
 Options:
-   -i,--installer PATH	Path to Endless OS installer image
+   -i,--installer PATH	Path to eosinstaller image
    -o,--os-image  PATH	Path to Endless OS image to add to installer
-   -l,--latest		Fetch latest OS and/or installer image
-   -h,--help		show this message
+   -l,--latest		Fetch latest OS and/or eosinstaller image
+
+If --latest is not specified, both --installer and --os-image are required.
+
 If --latest is specified and one or both of --installer or --os-image are
-missing, the latest image will be downloaded. If --latest is not specified,
-both --installer and --os-image must be specified.
+missing, the latest image will be downloaded.
 EOF
 }
 
 function check_exists() {
     if [ ! -f "$1" ]; then
-        echo "$1 does not exist or is not a file device"
+        echo "Error: $1 does not exist or is not a file"
```

Comment:
IMO, error messages should be redirected to `stderr` like `>&2`. I know you didn't write these messages originally, but it would be nice if you could fix them up.

Reply:
I've pushed a commit that redirects basically everything to `>&2`, which matches (eg) `eos-write-image` too.